frontend/components/Countdown.tsx

```diff
@@ -1,18 +1,26 @@
 import styles from '../styles/Home.module.css';
-import { compareAsc } from 'date-fns';
+import { compareAsc, getWeekYear } from 'date-fns';
 import CountDown from './Count.js';
 
-export default function Countdown() {
-  const current_date = new Date();
-  const targetDate = new Date('2023-11-22T00:00:00Z');
-  const variable = compareAsc(current_date, new Date(2023, 10, 5, 2, 0, 0));
+export default function Countdown({ date }: { date: Date }) {
+  const targetDate = new Date('2024-04-22T00:00:00Z');
+  const variable = compareAsc(date, targetDate);
+
   // today's date is after the date given by developer
   if (variable === 1) {
-    return (
-      <div className={styles.divider}>
-        <h2>Sign ups for 2023 have closed.</h2>
-      </div>
-    );
+    if (getWeekYear(targetDate) == 2023 && getWeekYear(date) >= 2024) {
```

Comment:
1. shouldn't this check if `targetDate` is 2024?
2. this could be improved by checking if `targetDate`'s year is equal to current year.
3. use triple `===` to enforce strict equality.